# Findings 
Here you find a summary of the initial findings

## Some stats

In [1]:
import pandas as pd
df = pd.read_csv('../data/archaeology_metadata.csv')
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 158254 entries, 0 to 158253
Data columns (total 13 columns):
 #   Column                                                Non-Null Count   Dtype  
---  ------                                                --------------   -----  
 0   dsPersistentId                                        158254 non-null  object 
 1   publicationStatus                                     158254 non-null  object 
 2   title                                                 158254 non-null  object 
 3   dsDescriptionValue                                    158254 non-null  object 
 4   dansSpatialPointX                                     56479 non-null   object 
 5   dansSpatialPointY                                     56479 non-null   object 
 6   dansSpatialPointScheme                                56304 non-null   object 
 7   dansSpatialBoxNorth                                   4432 non-null    object 
 8   dansSpatialBoxEast                          

In total, the archaeology data station contains 158254 published datasets. 
About 60000 have geospatial information in the metadata, either in the form of points (x, y), or a bounding box. 

--> About* 1/3 of the datasets have spatial metadata available. 

### Inspect the data with missing explicit geospatial metadata

In [2]:
df_nogeo = df[df.dansSpatialBoxNorth.isna() & df.dansSpatialPointX.isna()]

In [3]:
df_nogeo.info()

<class 'pandas.core.frame.DataFrame'>
Index: 98417 entries, 109 to 158234
Data columns (total 13 columns):
 #   Column                                                Non-Null Count  Dtype  
---  ------                                                --------------  -----  
 0   dsPersistentId                                        98417 non-null  object 
 1   publicationStatus                                     98417 non-null  object 
 2   title                                                 98417 non-null  object 
 3   dsDescriptionValue                                    98417 non-null  object 
 4   dansSpatialPointX                                     0 non-null      object 
 5   dansSpatialPointY                                     0 non-null      object 
 6   dansSpatialPointScheme                                4 non-null      object 
 7   dansSpatialBoxNorth                                   0 non-null      object 
 8   dansSpatialBoxEast                                    0 no

### Location files 
- This [location.xml](https://archaeology.datastations.nl/file.xhtml?fileId=3746676&version=1.0) is restricted

## Possibly useful variables 

| variable | description | example | source doi |
|---|---|---|---|
| dansTemporalSpatial:dansSpatialPoint | Has an X, Y and Scheme specification | 6.11557, 52.78667, longitude/latitude (degrees) |  10.17026/25AR/25AAV0BL | 
| dansTemporalSpatial:dansSpatialCoverageControlled |  | Netherlands | 10.17026/25AR/25AAV0BL | 
| dansTemporalSpatial:dansSpatialCoverageText | | 	"Sint-Michielsgestel", "NLD" | 10.17026/AR/KDIBWJ | 
| dansRelationMetadata:dansCollection:title | | Kalverstraat 11 Steenwijk | 10.17026/25AR/25AAV0BL | 
| ore:aggregates:schema:name | Contains the name of a file, often with location information | 115-261_Steenwijk_Kalverstraat 11.pdf / location.xml |  10.17026/25AR/25AAV0BL \ 10.17026/AR/KDIBWJ | 
| dansRelationMetadata:dansCollection:title | Title of the dataset, can contain clues on geospatial info |  Kalverstraat 11 Steenwijk | 10.17026/25AR/25AAV0BL | 
| citation:dsDescriptionValue | Contains a free description of the dataset | | | 
| ore:describes:language | Language of the report/dataset, might give a clue in toponym disambiguation step |  | | 




### Approach 1 to geospatial metadata enrichment: mapping place names/street addresses to coordinates 
